src/storeTvlInterval/getAndStoreTvl.ts

```diff
@@ -64,7 +63,12 @@ export async function storeTvl(
         continue;
       }
     }
-    await storeNewTvl(protocol, unixTimestamp, tvl);
+
+    const storeTokensAction = storeNewTokensValueLocked(protocol, unixTimestamp, tokensBalances);
```

Comment:
This would be called every hour, but it seems that you'd like to store daily values using it, should it be changed? I think that storing both daily and hourly may be interesting since we may want to use that data in the future for charts with small timeframes